Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN make build-static \
  && mkdir -p /opt/bin \
  && mv ./crowlet /opt/bin/crowlet
 
-FROM centurylink/ca-certs
+FROM golang:1.19-alpine
```

Comment:
Does this one include any CA? The previous one is getting very old indeed, I don't know if this image includes any.

Reply:
it looks like, I did the change because 0.2.1 was not working with recent Let's Encrypt certs.
I also just now listed the `/etc/ssl/certs/` folder inside built container on my local machine - it has some CA certs and I can confirm  that such build works with Let's Encrypt.